CriteoMoPubAdapter/Sources/CriteoMoPubAdapter/CRNativeAdDelegateHandler.h

```diff
@@ -0,0 +1,33 @@
+//
+//  CRNativeAdDelegateHandler.h
+//  CriteoMoPubAdapter
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+#import <CriteoPublisherSdk/CriteoPublisherSdk.h>
+
+@class CRNativeCustomEvent;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CRNativeAdDelegateHandler : NSObject <CRNativeLoaderDelegate>
```

Comment:
Same as previously commented I think. As this class just implement one protocol to be only used once by `CRNativeCustomEvent`, which requires exposing internals, why no just merging this into `CRNativeCustomEvent` ?

Reply:
Conflict on `delegate` between `MPNativeCustomEvent` and `MPNativeAdAdapter` preventing me from merging the two together.